CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [12.17.3]
+### Fixed
+- Stop emitting Dapp Viewed events on Firefox ([#32583](https://github.com/MetaMask/metamask-extension/pull/32583))
+- Make phishing warning events anonymous ([#32635](https://github.com/MetaMask/metamask-extension/pull/32635))
+- Keep showing the approve row in the batch transaction confirmation ([#32658](https://github.com/MetaMask/metamask-extension/pull/32658))
+- Support permit2 approvals in batch simulation ([#32733](https://github.com/MetaMask/metamask-extension/pull/32733))
+- Add metrics to check rejection of smart account update by user ([#32737](https://github.com/MetaMask/metamask-extension/pull/32737))
```

Comment:
Nit: This one is us fixing a metric event that we use internally. Not something with a user impact, should be safe to omit it here.

```suggestion
```